typescript/lib/parsers/openai.ts

```diff
@@ -173,6 +176,7 @@ export class OpenAIModelParser extends ParameterizedModelParser<CompletionCreate
       for (const choice of response.choices) {
         const output: ExecuteResult = {
           output_type: "execute_result",
+          // CompletionChoice does not support function calls
```

Comment:
Yes, this class is for the older OpenAI models that don't support function calling. Good comment to leave